### PR TITLE
Save IDs - Version-independent saves

### PR DESCRIPTION
**Pull request by Kaes3kuch3n**

### CODE REVIEW

**Review by inkompotato — COMMENTED** [review]
Awesome, Nice work.... I cannot test this, as my notebook is on its way to Lenovo, therefore I'd suggest, that someone else does the approve

File: Assets/Scripts/Managers/SaveManager.cs [hunk 1]
```diff
@@ -32,38 +33,51 @@ public void Save(string fileName)
...
                 save.campsites = new List<SavedCampsite>();
-                campsites.ForEach(cs => save.campsites.Add(new SavedCampsite(cs, cs.IsUnlocked)));
+                campsites.ForEach(c =>
+                {
+                    List<SavedBlueprint> blueprints = c.buildingBlueprints.Select(b => new SavedBlueprint
+                    {
+                        blueprintId = b.gameObject.GetId(),
+                        buildingId = b.Building.gameObject.GetId(),
+                        blueprintActive = b.IsActive,
+                        buildingActive = b.Building.IsBuilt
```
Comment (SamuelllG) [inline Assets/Scripts/Managers/SaveManager.cs:73]: I don't think you really need two variables for this.

File: Assets/Scripts/Managers/SaveManager.cs [hunk 2]
```diff
@@ -78,80 +92,152 @@ public void Save(string fileName)
...
+                    {
+                        building.IsBuilt = true;
+                        blueprint.gameObject.GetComponentInChildren<BlueprintTooltip>().gameObject.SetActive(false);
+                    }
+                    else if (savedBlueprint.blueprintActive && !savedBlueprint.buildingActive)
+                    {
+                        building.IsBuilt = false;
+                    }
                 });
+                if (savedCampsite.isUnlocked && !campsite.IsUnlocked)
```
Comment (SamuelllG) [inline Assets/Scripts/Managers/SaveManager.cs:176]: You could check this before going through the savedBlueprints

**Review by SamuelllG — DISMISSED** [review]

**Comment by SamuelllG** [comment]
I guess we have more important stuff to do than pay attention to the little things

**Review by DavidPgl — APPROVED** [review]

**Review by SamuelllG — APPROVED** [review]

**Pull request merged**